Elevel/Domain/Models/Test.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Elevel.Domain.Models
+{
+    public class Test: BaseDataModel
+    {
+        public Guid UserId { get; set; }
+        public User User { get; set; }
+
+        public Guid? HrId { get; set; }
+        public User Hr { get; set; }
+
+        public Guid? CoachId { get; set; }
+        public User Coach { get; set; }
+
+        public string Level { get; set; }
```

Comment:
I think it's good to add LevelEnum to avoid strings. String is usual very error prone